doc/mgr/dashboard.rst

```diff
@@ -37,35 +37,44 @@ various aspects of your Ceph cluster:
   accounts with different permissions (roles). The user accounts and roles
   can be modified on both the command line and via the WebUI.
   See :ref:`dashboard-user-role-management` for details.
+* **Single Sign-On (SSO)**: the dashboard supports authentication
+  via an external identity provider using the SAML 2.0 protocol. See
+  :ref:`dashboard-sso-support` for details.
 * **SSL/TLS support**: All HTTP communication between the web browser and the
   dashboard is secured via SSL. A self-signed certificate can be created with
   a built-in command, but it's also possible to import custom certificates
   signed and issued by a CA. See :ref:`dashboard-ssl-tls-support` for details.
-* **Overall cluster health**: Displays the overall cluster status, storage
-  utilization (e.g. number of objects, raw capacity, usage per pool), a list of
-  pools and their status and usage statistics.
+* **Auditing**: the dashboard backend can be configured to log all PUT, POST
+  and DELETE API requests in the Ceph audit log. See :ref:`dashboard-auditing`
+  for instructions on how to enable this feature.
+* **Internationalization (I18N)**: use the dashboard in different languages.
+* **Overall cluster health**: Displays overall cluster status, performance
+  and capacity metrics.
 * **Cluster logs**: Display the latest updates to the cluster's event and audit
   log files.
 * **Hosts**: Provides a list of all hosts associated to the cluster, which
   services are running and which version of Ceph is installed.
 * **Performance counters**: Displays detailed service-specific statistics for
   each running service.
 * **Monitors**: Lists all MONs, their quorum status, open sessions.
-* **Configuration Reference**: Lists all available configuration options,
-  their description and default values.
+* **Configuration Editor**: View all available configuration options,
+  their description, type and default values and edit the current values.
 * **Pools**: List all Ceph pools and their details (e.g. applications, placement
```

Comment:
```suggestion
* **Pools**: Lists all Ceph pools and their details (e.g. applications, placement
```
(Just a small suggestion because "Lists" is used for all mentioned items below)